docs/tutorials/03_chapter/loading_model_from_oneflowhub.md

```diff
@@ -0,0 +1,296 @@
+```
+
+#### Base64 结果
+
+用于 API 服务。 有关详细信息，请参阅 [#2291](https://github.com/ultralytics/yolov5/pull/2291) 和 [Flask REST API](https://github.com/ultralytics/yolov5/tree/master/utils/flask_rest_api) 示例。
+
+```python
+results = model(im)  # inference
+
+results.ims # array of original images (as np array) passed to model for inference
+results.render()  # updates results.ims with boxes and labels
+for im in results.ims:
+    buffered = BytesIO()
+    im_base64 = Image.fromarray(im)
+    im_base64.save(buffered, format="JPEG")
+    print(base64.b64encode(buffered.getvalue()).decode('utf-8'))  # base64 encoded image with results
+```
+
+#### 裁剪结果
+
+返回的检测结果可以被裁剪：
+
+```python
+results = model(im)  # inference
+crops = results.crop(save=True)  # cropped detections dictionary
+```
+
+#### Pandas 结果
+
+结果可以作为[Pandas DataFrames](https://pandas.pydata.org/)返回：
+
+```python
+results = model(im)  # inference
+results.pandas().xyxy[0]  # Pandas DataFrame
+```
+
+<details>
+  <summary>Pandas输出（点击展开）</summary>
+  <pre><code> 
+    print(results.pandas().xyxy[0])
+    xmin        ymin         xmax        ymax  confidence  class    name
+    0  743.290649   48.343842  1141.756348  720.000000    0.879861      0  person
+    1  441.989624  437.336670   496.585083  710.036255    0.675118     27     tie
+    2  123.051117  193.237976   714.690674  719.771362    0.666694      0  person
+    3  978.989807  313.579468  1025.302856  415.526184    0.261517     27     tie
+  </code></pre>
+</details>
+
+#### 排序后的结果
+
+结果可以按列排序，例如从左到右（x轴）对车牌数字检测结果进行排序：
+
+```python
+results = model(im)  # inference
+results.pandas().xyxy[0].sort_values('xmin')  # sorted left-right
+```
+
+#### Box-Cropped 结果
+
+结果可以返回并保存为 detection crops：
+
+```python
+results = model(im)  # inference
+crops = results.crop(save=True)  # cropped detections dictionary
+```
+
+#### JSON 结果
+
+结果一旦使用 `.pandas` 被保存为 pandas 数据格式，就可以再使用 `.to_json()` 方法保存为 JSON 格式。可以使用 `orient` 参数修改 JSON 格式。请查看 pandas 的 `.to_json()` 方法的[文档](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_json.html)了解细节。
+
+```python
+results = model(ims)  # inference
+results.pandas().xyxy[0].to_json(orient="records")  # JSON img1 predictions
+```
+
+<details>
+  <summary>Json输出（点击展开）</summary>
+  <pre><code> 
+    [{"xmin":743.2906494141,"ymin":48.3438415527,"xmax":1141.7563476562,"ymax":720.0,"confidence":0.87986058,"class":0,"name":"person"},{"xmin":441.9896240234,"ymin":437.3366699219,"xmax":496.5850830078,"ymax":710.0362548828,"confidence":0.6751183867,"class":27,"name":"tie"},{"xmin":123.0511169434,"ymin":193.2379760742,"xmax":714.6906738281,"ymax":719.7713623047,"confidence":0.6666944027,"class":0,"name":"person"},{"xmin":978.9898071289,"ymin":313.5794677734,"xmax":1025.3028564453,"ymax":415.526184082,"confidence":0.2615173161,"class":27,"name":"tie"}]
+  </code></pre>
+</details>
+
+
+#### 自定义模型
+
+这个例子展示使用 OneFlow Hub 加载一个自定义的在VOC数据集上进行训练的20个类别的 YOLOV5s 模型 `best.pt` 。
```

Comment:
```suggestion
这个例子展示使用 OneFlow Hub 加载一个自定义的在VOC数据集上进行训练的20个类别的 YOLOV5s 模型 `best` 。
```python
model = oneflow.hub.load('Oneflow-Inc/one-yolov5', 'custom', path='path/to/best') # local model
model = oneflow.hub.load('/path/to/one-yolov5', 'custom', path='path/to/best') # local repo
```
```